Source/src/components/ComponentMeshRenderer.cpp

```diff
@@ -18,13 +18,12 @@
 
 #include <debugdraw.h>
 
-Hachiko::ComponentMeshRenderer::ComponentMeshRenderer(GameObject* container) 
-    : Component(Type::MESH_RENDERER, container)
+Hachiko::ComponentMeshRenderer::ComponentMeshRenderer(GameObject* container) :
+    Component(Type::MESH_RENDERER, container)
```

Comment:
To be honest, I find this initializer list style inferior to the previous one, as the previous looks like the following when there are many members:

```
Namespace::Class::Class() 
    : Base()
    , some_member(2)
    , another_member(0.0f)
    , yet_another_member(false) {}
    
```

Is it possible that we define this in our clang coding style?

Reply:
the format that it has is defined, I just ran the formatter on some files, we can change that directly on the config file, but that is outside the scope of this